README.md

```diff
@@ -61,7 +61,7 @@ Example usage:`./test/generate_incident.sh ClusterHasGoneMissing 2b94brrrrrrrrrr
 2) Create a payload file containing the incident ID
   ```bash
   export INCIDENT_ID=
-  echo '{"event": {"data":{"id": "${INCIDENT_ID}"}}}' > ./payload
```

Comment:
This was the wrong format. The new format matches what we would get if we would add a default webhook to a service orchestration.